buildkite/graphql/pipeline.graphql

```diff
@@ -52,60 +54,17 @@ query getPipeline($slug: ID!) {
     }
 }
 
+# @genqlient(for: "PipelineCreateInput.branchConfiguration", pointer: true)
 # @genqlient(for: "PipelineCreateInput.clusterId", pointer: true)
-# @genqlient(for: "PipelineCreateInput.visibility", omitempty: true)
-# @genqlient(for: "PipelineCreateInput.pipelineTemplateId", omitempty: true)
 # @genqlient(for: "PipelineCreateInput.nextBuildNumber", omitempty: true)
+# @genqlient(for: "PipelineCreateInput.pipelineTemplateId", omitempty: true)
+# @genqlient(for: "PipelineCreateInput.visibility", omitempty: true)
```

Comment:
++ `# @genqlient(for: "PipelineCreateInput.defaultTimeoutInMInutes", pointer: true)`

Reply:
Same on Update